DependencyInjection/Configuration.php

```diff
@@ -1015,10 +1016,13 @@ private function addVerbosityLevelSection(ArrayNodeDefinition $handerNode)
                         ->then(function ($v) {
                             $map = [];
                             $verbosities = ['VERBOSITY_QUIET', 'VERBOSITY_NORMAL', 'VERBOSITY_VERBOSE', 'VERBOSITY_VERY_VERBOSE', 'VERBOSITY_DEBUG'];
-                            // allow numeric indexed array with ascendning verbosity and lowercase names of the constants
+                            $verbosityConstants = [OutputInterface::VERBOSITY_QUIET, OutputInterface::VERBOSITY_NORMAL, OutputInterface::VERBOSITY_VERBOSE, OutputInterface::VERBOSITY_VERY_VERBOSE, OutputInterface::VERBOSITY_DEBUG];
```

Comment:
The array should look like `[OutputInterface::VERBOSITY_QUIET => 'VERBOSITY_QUIET', ...]` to remove the `array_search` and make the code more readable.

Reply:
I think we can make the code a little clearer with this changes: https://github.com/HeahDude/monolog-bundle/pull/1